@here/harp-omv-datasource/lib/OmvData.ts

```diff
@@ -185,15 +185,13 @@ export class GeometryCommands {
         extent: number = 4096,
         visitor: GeometryCommandsVisitor
     ) {
-        const longitudeScale = geoBox.longitudeSpan / extent;
-
         const geometryCount = geometry.length;
 
         const { north, west } = geoBox;
-
         const N = Math.log2(extent);
-
+        const scale = Math.pow(2, tileKey.level + N);
```

Comment:
Shouldn't that code be in the tiling scheme?

Reply:
yes but unfortunately our TileKey/TilingScheme doesn't handle tiles with level greater than 23. So we won't be able to represent anything with level 15+